src/dlgProfilePreferences.cpp

```diff
@@ -538,6 +540,8 @@ void dlgProfilePreferences::enableHostDetails()
 
     groupBox_doubleClick->setEnabled(true);
 
+    checkBox_autoWrap->setEnabled(true);
+
```

Comment:
```suggestion

```

Reply:
Forget the previous change I suggested - again I didn't see the whole thing correctly.